packages/slate-editor/src/extensions/embed/components/EmbedMenu.tsx

```diff
@@ -107,6 +120,16 @@ export function EmbedMenu({
                 </Toolbox.Section>
             )}
 
+            <Toolbox.Section caption="Change to...">
+                <OptionsGroup
+                    name="presentation"
+                    options={PRESENTATION_OPTIONS}
+                    selectedValue={EmbedNode.Presentation.EMBED}
+                    onChange={(presentation) => onTransform(presentation)}
```

Comment:
Is there something preventing from doing this?

```suggestion
                    onChange={onTransform}
```